src/Microsoft.Graph.Cli.Core/Authentication/AuthenticationServiceFactory.cs

```diff
@@ -24,6 +24,8 @@ public virtual async Task<ILoginService> GetAuthenticationServiceAsync(Authentic
         {
```

Comment:
why do we need two different methods here since the implementation is essentially the same?

Reply:
sorry, disregard this, I misread something